templates/solutions/infrastructure/index.html

```diff
@@ -175,12 +171,10 @@ <h2>Full-stack automation</h2>
         <h2>One vendor, one subscription for security and support</h2>
       </div>
       <div class="col">
-        <p class="p-section--shallow">Say goodbye to unpredictable costs. Canonical offers you the convenience of a
-          single subscription for your platform’s security maintenance, support and compliance: Ubuntu Pro.</p>
+        <p class="p-section--shallow">Say goodbye to unpredictable costs. Canonical offers you the convenience of a single subscription for your platform’s security maintenance, support and compliance: Ubuntu Pro</p>
```

Comment:
Should this have a full stop?

Reply:
Yes as per the copy doc. Only headings and bullet points with a single sentence should _not_ have a period at the end.